sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs

```diff
@@ -0,0 +1,776 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal static class TextAnalyticsServiceSerializer
```

Comment:
Most likely, we will need support for `System.Text.Json`. So instead of huge static serializer, we can make individual converters that are derived from `JsonConverter<T>`.

Reply:
#8711